src/components/MultiComboBox/useListbox.tsx

```diff
@@ -0,0 +1,316 @@
+          if (activeOptionIndex === null) {
+            return
+          }
+          const activeOption = options[activeOptionIndex]
+          if (activeOption.isAdding) {
+            onAdd && onAdd(activeOption.label)
+            return
+          }
+          if (activeOption && !activeOption.disabled) {
+            const { value, label } = activeOption
+            onSelect({ value, label })
+          }
+          break
+        }
+      }
+    },
+    [activeOptionIndex, moveActiveOptionIndex, onAdd, onSelect, options],
+  )
+
+  const listboxId = useId()
+  const addingButtonId = useId()
+  const optionIdPrefix = useId()
+  const getOptionId = useCallback(
+    (option: Option) => {
+      if (option.isAdding) {
+        return addingButtonId
+      }
+      return `${optionIdPrefix}-${option.label}`
+    },
+    [addingButtonId, optionIdPrefix],
+  )
+  const activeDescendant = (() => {
+    if (activeOptionIndex === null) {
+      return undefined
+    }
+    const activeOption = options[activeOptionIndex]
+    if (activeOption) {
+      return getOptionId(activeOption)
+    }
+    return undefined
+  })()
+
+  const theme = useTheme()
+  const { portalRoot } = usePortal()
+  const listboxRef = useRef<HTMLDivElement>(null)
+
+  const renderListbox = () => {
+    return createPortal(
+      <Container
+        {...dropdownStyle}
+        themes={theme}
+        id={listboxId}
+        className={isExpanded ? undefined : 'hidden'}
+        ref={listboxRef}
+        role="listbox"
+      >
+        {options.map((option, i) => {
+          const activeClass = activeOptionIndex === i ? 'active' : undefined
+          const { label, disabled, isAdding } = option
+          if (isAdding) {
+            return (
+              <AddButton
+                key={`add-${label}`}
+                themes={theme}
+                onClick={() => onAdd && onAdd(label)}
+                onMouseOver={() => setActiveOptionIndex(0)}
+                className={activeClass}
+                id={addingButtonId}
+                role="option"
+              >
+                <AddIcon
+                  name="fa-plus-circle"
+                  size={14}
+                  color={theme.palette.TEXT_LINK}
+                  $theme={theme}
+                />
+                <AddText themes={theme}>「{label}」を追加</AddText>
+              </AddButton>
+            )
+          }
+          return (
+            <SelectButton
+              key={`item-${label}`}
+              type="button"
+              themes={theme}
+              disabled={disabled}
+              onClick={() => onSelect(option)}
+              onMouseOver={() => setActiveOptionIndex(i)}
+              className={activeClass}
+              id={getOptionId(option)}
+              role="option"
+            >
+              {label}
+            </SelectButton>
+          )
+        })}
+
+        {isDuplicated && (
+          <NoItems themes={theme} aria-live="polite">
```

Comment:
`role="alert"` を追加しても良さそうですー。 `role="alert"` は暗黙の `aria-live="assertive"` を持ちますが、`aria-live="polite"` で上書きしても問題ありません。ちなみに `aria-live="assertive"` ではない理由は、それほど深刻なエラーではないという感じでしょうか？

```suggestion
          <NoItems themes={theme} role="alert" aria-live="polite">
```

Reply:
ありがとうございます！
`polite` にした理由としては、これらはinput入力値に連動するメッセージなので `assertive` だと少しうるさすぎるように感じたためです！